spe1/SPE1CASE1_WATER.DATA

```diff
@@ -0,0 +1,252 @@
+-- This reservoir simulation deck is made available under the Open Database
+-- License: http://opendatacommons.org/licenses/odbl/1.0/. Any rights in
+-- individual contents of the database are licensed under the Database Contents
+-- License: http://opendatacommons.org/licenses/dbcl/1.0/
+
+-- Copyright (C) 2019 SINTEF
+-- Copyright (C) 2020 Equinor
```

Comment:
Maybe there should be a TNO copyright here? (Depends on contract terms and conditions covering your work.)

Reply:
My understanding is that tasks sponsored by Equinor should have Equinor copyright. When the task is sponsored by other TNO projects, we add TNO copyright. Is that correct, @alfbr?

Reply:
That is correct.